docs/_docs/posts.md

```diff
@@ -170,24 +171,27 @@ a basic example of how to create a list of posts from a specific category.
 
 First, in the `_layouts` directory create a new file called `category.html` - in
 that file put (at least) the following:
-```
+
+{% raw %}
+```liquid
 ---
 layout: page
 ---
+
 {% for post in site.categories[page.category] %}
-    <a href="{{ post.url | prepend: site.baseurl }}">
+    <a href="{{ post.url | absolute_url }}">
```

Comment:
This makes it `| prepend: site.baseurl | prepend: site.url` which is technically a change in behaviour. Just noting it. 🤷‍♂️

Reply:
True, I thought we should encourage the use of the filter in our docs.  Should we?
There is no URLs good practices in http://ben.balter.com/jekyll-style-guide/ yet but it might be a good candidate.